flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/validation/ScalarFunctionsValidationTest.scala

```diff
@@ -78,10 +78,21 @@ class ScalarFunctionsValidationTest extends ScalarTypesTestBase {
   }
 
   @Test(expected = classOf[ValidationException])
-  def testTimestampAddWithWrongQuantity(): Unit ={
+  def testTimestampAddWithWrongQuantity(): Unit = {
     testSqlApi("TIMESTAMPADD(YEAR, 1.0, timestamp '2016-02-24 12:42:25')", "2016-06-16")
   }
 
+  @Test(expected = classOf[ValidationException])
+  def testLpadWithNull(): Unit = {
+    // Must fail. Parameter of base string must not be null.
+    testSqlApi("LPAD(null,1,'??')", "")
```

Comment:
I think we don't need to fail here. Returning null should be the SQL standard here.